concepts/eventing-code-guidelines.md

```diff
@@ -0,0 +1,1147 @@
+# Eventing team code guidelines
+
+**Document Intention** 
+
+The intention of this document is to provide style, testing, and code guidelines for components of the **eventing** team. Other teams in Kyma may follow different guidelines.
+The document is supposed to be a **living** document that can be changed by anyone in the team. However the team majority (50%) must agree on changes for this guideline. 
+
+This guide shall be publicly available so that it can be referenced when reviewing pull requests. The agreements of this guide shall not affect **external collaborators**. If an external PR does not align with our guidelines, we should accept the PR as it is and apply the guidelines ourselves in a follow-up PR so we don't block external contributors. Of course, also external PRs shall meet our quality standards but we shouldn't be too nitpicky.
```

Comment:
Who are the external collaborators? Non-Tunas or non-Kyma contributors?

Reply:
As discussed internally: we leave it as it is. 
It means external people who are not working primarily for Kyma.